openage/convert/main.py

```diff
@@ -205,6 +225,10 @@ def flag(name):
 
         converted_count += 1
 
+    # save the asset location for next time
+    with asset_location_path.open("w") as file_obj:
+        file_obj.write(data_dir.resolve_native_path().decode('utf-8'))
```

Comment:
I think we should write it with `wb`, then the decode is not necessary and we can ignore if the native path is valid utf8. (Above, also read with `wb` and try decoding with utf8 like currently done).